scripts/generate_token.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import asyncio
+import contextlib
+import sys
+import os
+
+from fastapi_users.authentication.strategy import JWTStrategy
+from sqlalchemy.future import select
+
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
+
+from alws.dependencies import get_db
+from alws.auth.dependencies import get_user_db
+from alws.auth.user_manager import get_user_manager
+from alws.models import User
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-e', '--email', required=False, type=str,
+                        default=None,
+                        help='User e-mail')
+    parser.add_argument('-u', '--username', required=False, type=str,
+                        default=None,
+                        help='User e-mail')
+    parser.add_argument('-s', '--secret', required=True, type=str,
+                        help='JWT secret')
+    return parser.parse_args(sys.argv[1:])
```

Comment:
there is no need to pass args from `sys.argv`, `argparse` module will do it for you
https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.parse_args